Bottom: 1fbb77616bb93b72ebcf06c6b1fd530b2f477990
Top:    448ae10f6f9d3af9e92c6784852f117530e3ad8b
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:24:18 +0800

Refresh of net-core-introduce-xdp-rx

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index 0d281e4..af8ca96 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -1171,7 +1171,7 @@ static int macvlan_port_create(struct net_device *dev)
 	skb_queue_head_init(&port->bc_queue);
 	INIT_WORK(&port->bc_work, macvlan_process_broadcast);
 
-	err = netdev_rx_handler_register_xdp(dev, macvlan_handle_frame, port);
+	err = netdev_rx_handler_register(dev, macvlan_handle_frame, port);
 
 	if (err)
 		kfree(port);
